app/controllers/gi_controller.rb

```diff
@@ -11,6 +11,12 @@ def client
   end
 
   def scrubbed_params
-    params.except(:action, :controller, :format)
+    params.except(:action, :controller, :format).transform_values do |v|
+      begin
+        v.respond_to?(:encode) && v.encode!('UTF-8', 'binary')
+      rescue StandardError
+        raise Common::Exceptions::InvalidFieldValue.new('parameter', v.scrub)
```

Comment:
Is there a better error we could rescue from than StandardError? StandardError is so generic, I wonder if maybe we will get into some confusing debugging situations.